### PR TITLE
Added image testing instructions to README.

### PR DESCRIPTION
**Pull request by markdorison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,15 @@ Then, push each of the newly generated Docker images to the local registry with
...
 
 See https://github.com/docker/distribution/releases for the latest version of the registry server
+
+## Testing Images
+If you have created or modified an image, and have already created a local
+Docker repository as described above, perform the following steps to test your
+changes:
+
+1. `make <imagename>`
+1. `make push <imagename>`
```
Comment (q0rban) [inline README.md]: Is this step necessary for testing an image @bchavet ?

**Comment by q0rban** [comment]
Awesome!

File: README.md (hunk 1 above)
Comment (bchavet) [reply README.md]: Not the `make push`, no. after `make <imagename>` it already exists in your local docker images as `localhost:5000/imagename`

Comment (markdorison) [reply README.md]: Just for my own knowledge, when _is_ `make push` useful/needed?

Comment (bchavet) [reply README.md]: I use it to deploy to our production registry.

**Review by bchavet — APPROVED** [review]

**Pull request merged**